bench/playbooks/roles/nodejs/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-node_version: 12
+node_version: 14
```

Comment:
Does this work with Frappe v12 installs?

Reply:
Will test

Reply:
@gavindsouza 

I've left it as 12 as central runs v13 and v12 both with this.